src/libcollections/dlist.rs

```diff
@@ -304,6 +304,19 @@ impl<T> DList<T> {
     /// Returns `true` if the `DList` is empty.
     ///
     /// This operation should compute in O(1) time.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
```

Comment:
We've been leaving off the 'rust' in these examples. It defaults to `rust`, but it's nice to have when the code blocks are by themselves, ie, in the longform docs. But since this is embedded inside of a Rust comment, Github doesn't highlihght them anyway, so it's just noise.